features/admin.claims.v1/components/edit/local-claim/edit-additional-properties-local-claims.tsx

```diff
@@ -79,9 +85,43 @@ export const EditAdditionalPropertiesLocalClaims:
             [ featureConfig, allowedScopes ]
         );
 
+        const RESTRICTED_PROPERTY_KEYS: string[] = [ "isUnique" ];
+
+        const [ showWarning, setShowWarning ] = useState<boolean>(false);
+
+        const { UIConfig } = useUIConfig();
+
         return (
             <EmphasizedSegment>
                 <Grid>
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 16 }>
+                            { showWarning && UIConfig?.isClaimUniquenessValidationEnabled && (
+                                <Message
+                                    type="warning"
+                                    content={ (
+                                        <>
+                                            The &apos;isUnique&apos; property is deprecated. Please use the &nbsp;
+                                            <Link
+                                                external={ false }
+                                                onClick={ () => {
+                                                    history.push({
+                                                        pathname: AppConstants.getPaths()
+                                                            .get("LOCAL_CLAIMS_EDIT")
+                                                            .replace(":id", claim.id)
+                                                    });
+                                                } }
+                                            >
+                                                Uniqueness Validation Dropdown
+                                            </Link>
+                                                &nbsp; to configure claim uniqueness.
```

Comment:
Let's use i18n here to provide translation support.
Ref: https://github.com/wso2/identity-apps/blob/master/features/admin.applications.v1/components/my-account/my-account-overview.tsx#L72

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/468ea6e0d5d0020a1e229e8e8d758ef8b3c9f2f2